client/mainmenu/CMainMenu.cpp

```diff
@@ -469,18 +469,26 @@ CMultiMode::CMultiMode(ESelectionScreen ScreenType)
 	background = std::make_shared<CPicture>(ImagePath::builtin("MUPOPUP.bmp"));
 	pos = background->center(); //center, window has size of bg graphic
 
-	picture = std::make_shared<CPicture>(ImagePath::builtin("MUMAP.bmp"), 16, 77);
+	const auto& multiplayerConfig = CMainMenuConfig::get().getConfig()["multiplayer"];
+	if (multiplayerConfig.isVector() && !multiplayerConfig.Vector().empty())
+		picture = std::make_shared<CPicture>(ImagePath::fromJson(*RandomGeneratorUtil::nextItem(multiplayerConfig.Vector(), CRandomGenerator::getDefault())), 16, 77);
+
+	if (multiplayerConfig.isString())
+		picture = std::make_shared<CPicture>(ImagePath::fromJson(multiplayerConfig), 16, 77);
```

Comment:
Probably needs fallback to `MuMap` if this key is not present - not everyone will update every mod on new release

Reply:
I was thinking about that fallback too. But I have removed it and prepared update for HotA, WoG and H3 themes pack instead, as all of these mods have broken loading bar.

Reply:
Not everyone updates mods ASAP, so game should work even if player have not update mod, especially for hotfixes.

You can mark it with `//mods compatibilty for 1.6.3` comment, and remove it in 1.7+

Reply:
True, we’ll re-add it. Maybe fallbacks for backgrounds should be added and keep for case when certain element doesn’t exist in json?

Reply:
Which backgrounds? This "mumap"?
IMO fallback is only needed to give some time for mods to update & for players to install update. Error message to log would also be a good idea btw.

Mods should generally provide all elements in cases like this one, if not - then this should be viewed as bug in a mod. Keeping fallback for long time may be useful, but only to avoid crashes.

Reply:
Mumps fallback added. Will remove it in MainMenu PR for 1.7